backend/api/urls.py

```
@@ -8,7 +8,7 @@
 )
 from .admin_views import (
     UserSearchView, PromoteToAdminView, VideoManagementView, 
-    VideoStatsView
+    VideoStatsView, WebcamRecordingsView
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)